mwdb/core/config.py

```diff
@@ -23,14 +24,30 @@ def path(v) -> Optional[str]:
     return v
 
 
+class StorageProviderType(Enum):
+    DISK = "DISK"
+    BLOB = "BLOB"
+
+
+def storage_provider_from_str(v: str) -> Optional[StorageProviderType]:
+    if not v:
+        return None
+    
+    v = v.upper()
+    try:
+        return StorageProviderType[v]
+    except KeyError:
+        raise ValueError(f"Blob Storage Provider {v} doesn't exist")
+        raise ValueError(f"Blob Storage Provider {v} doesn't exist")
```

Comment:
just noticed, duplicate return

```suggestion
        raise ValueError(f"Blob Storage Provider {v} doesn't exist")
```